samples/snippets/create_kmeans_model_test.py

```diff
@@ -0,0 +1,74 @@
+def test_kmeans_sample():
+    import datetime
```

Comment:
You're missing "region tags" so that this code sample can be embedded in the documentation.

Let's use

```
# [START bigquery_dataframes_bqml_kmeans]
...
# [END bigquery_dataframes_bqml_kmeans]
```

Reply:
Added region tags for start and end.